### DISCUSSION TITLE
Qute-pass fails and password_fill works for some sites only [both for gnu application 'pass']

### DISCUSSION THREAD

**Discussion by daUnknownCoder** [qutebrowser/qutebrowser#7795] [Q&A] [answered]
the userscript qute-pass failed for me like :
Status : Userscript exited with status 3.
Standard Error : Failed to match username, target: path, error: Pattern did not match target

i made the userscript password_fill work for me but it was only successful for a few sites like : hoyoverse.com, google.com fontstruct.com id.unity.com
and unsuccessful for others like : [www.reddit.com](http://www.reddit.com/) [www.twitch.tv](http://www.twitch.tv/) hoyolab.com spotify.com

so what do i do... 

Now i dont know how to use keepassx + i dont use kde [i use bspwm] and i installed keepassx but i dont know how to import my csv to keepassx, i want no-money options....

**Comment by bananabr34d** [comment, marked as answer]
What naming convention are you using in pass?  For the script to work you must follow:

`reddit.com/<username>`
